resources/js/visual-editor/blocks/image/image.css

```
@@ -0,0 +1,628 @@
+.wp-block-image.alignleft > figcaption,
+.wp-block-image.alignright > figcaption,
+.wp-block-image.aligncenter > figcaption,
+.wp-block-image .alignleft > figcaption,
+.wp-block-image .alignright > figcaption,
+.wp-block-image .aligncenter > figcaption {
+    display: table-caption;
+    caption-side: bottom;
+}
+
+.wp-block-image .alignleft {
+    /*rtl:ignore*/
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)